packages/dynamite/dynamite_runtime/lib/src/utils/codecs.dart

```diff
@@ -0,0 +1,132 @@
+import 'dart:convert';
+
+import 'package:built_collection/built_collection.dart';
+import 'package:built_value/serializer.dart';
+import 'package:dynamite_runtime/models.dart';
+
+/// A [HeaderCodec] encodes the json representation of a `Header` objects to
+/// strings and decodes strings to serialized `Header` objects.
```

Comment:
```suggestion
/// A [HeaderCodec] encodes the json representation of a [Header] objects to
/// strings and decodes strings to serialized [Header] objects.
```

Reply:
The Header type is not defined in the scope of this file.

Reply:
It imports the models file so it should be?

Reply:
My bad sorry.
I didn't remember that I changed this last night :sweat_smile:

Reply:
np